mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -1685,6 +1685,9 @@ static constexpr std::array kExplicitAttributes{
     StringLiteral("target-features"),
     StringLiteral("unsafe-fp-math"),
     StringLiteral("vscale_range"),
+    StringLiteral("denormal-fp-math"),
+    StringLiteral("denormal-fp-math-f32"),
+    StringLiteral("fp-contract"),
```

Comment:
nit: Can you sort the array alphabetically. It looks like it was sorted before.

Reply:
Yeah, done